src/v/redpanda/application.cc

```diff
@@ -1806,6 +1806,9 @@ void application::start_runtime_services(
     syschecks::systemd_message("Starting controller").get();
     controller->start(cd, app_signal.abort_source()).get0();
 
+    syschecks::systemd_message("Starting node isolation watcher").get();
+    _node_isolation_watcher->start().get();
```

Comment:
Perhaps we should start this after RPC?  We don't really want to start possibly considering the node isolated until we are at least able to talk to  our peers

Reply:
I have a question: health_monitor is part of controller. And it started before rpc and uses it to collect cluster health, right?